blocks/medium-manual-promo-block/features/medium-manual-promo/default.test.jsx

```diff
@@ -107,7 +107,7 @@ describe("the medium promo feature", () => {
 		};
 		const wrapper = mount(<MediumManualPromo customFields={noHeadline} />);
```

Comment:
This test is not using the correct data 
```suggestion
		const wrapper = mount(<MediumManualPromo customFields={noImage} />);
```

Reply:
My bad - looks like I got confused looking at the GitHub diff

Reply:
We can undo this change